go.mod

```diff
@@ -5,7 +5,7 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/didip/tollbooth v4.0.2+incompatible
 	github.com/drewolson/testflight v1.0.0
-	github.com/dusk-network/bls12_381-sign-go v0.1.3
+	github.com/dusk-network/bls12_381-sign v0.1.0-merge2
```

Comment:
Shouldn't we have here `master` instead of a temp tag?

Reply:
I don't think the master branch is the same as fix-33 because we were waiting to confirm all of this. So when master branch is also the merged together ipc/cgo combo version it would be changed to `latest` and it would grab from the master branch.

Reply:
https://github.com/dusk-network/bls12_381-sign/pull/34 I have noted that we should merge that PR before this one so as to make the new merged repository the master/release state of it. Then this repo's go.mod should be amended to the latest on the other repo now master branch and this one can then be merged and closed.